editor/components/post-type-support-check/index.js

```diff
@@ -15,8 +15,10 @@ import { compose } from '@wordpress/element';
  */
 import { getCurrentPostType } from '../../store/selectors';
 
-function PostTypeSupportCheck( { postType, children, supportKey } ) {
-	const isSupported = get( postType, [ 'data', 'supports', supportKey ], false );
+function PostTypeSupportCheck( { postType, children, supportKeys } ) {
+	const isSupported = some(
```

Comment:
Should this be `every`? I could imagine a need for both cases, i.e. a component that requires post type support for both X _and_ Y, but alternatively X _or_ Y.

Reply:
I hesitated on adding an "operator". For now we only have the "or" use-case.